nvidia-settings.json.in

```diff
@@ -0,0 +1,94 @@
+{
+	"id":"com.nvidia.nvidia-settings",
+	"runtime": "org.freedesktop.Platform",
+	"branch":"@@NVIDIA_VERSION@@",
+	"runtime-version": "1.6",
+	"sdk":"org.freedesktop.Sdk",
+	"command":"nvidia-settings",
+	"finish-args":[
+		"--device=dri",
+		"--socket=x11",
+		"--share=ipc",
+		"--extension=org.freedesktop.Platform.GL=version=1.4",
+		"--extension=org.freedesktop.Platform.GL=versions=1.6;1.4",
+		"--extension=org.freedesktop.Platform.GL=directory=lib/GL",
+		"--extension=org.freedesktop.Platform.GL=subdirectories=true",
+		"--extension=org.freedesktop.Platform.GL=no-autodownload=true",
+		"--extension=org.freedesktop.Platform.GL=autodelete=false",
+		"--extension=org.freedesktop.Platform.GL=add-ld-path=lib",
+		"--extension=org.freedesktop.Platform.GL=merge-dirs=vulkan/icd.d;glvnd/egl_vendor.d",
+		"--extension=org.freedesktop.Platform.GL=download-if=active-gl-driver",
+		"--extension=org.freedesktop.Platform.GL=enable-if=active-gl-driver"
+	],
+	"cleanup":[
+		"/include",
+		"*.a",
+		"*.la",
+		"*.lo",
+		"/lib/pkgconfig",
+		"/lib/debug",
+		"/share/gtk-doc",
+		"/share/man",
+		"/share/gtk-2.0",
+		"/lib/gtk-2.0/include"
+	],
+	"cleanup-commands":[
+		"mkdir -p /app/lib/GL"
+	],
+	"rename-icon":"nvidia-settings",
+	"rename-desktop-file":"nvidia-settings.desktop",
+	"modules":[
+		{
+			"name":"nvidia-settings",
+			"no-autogen":true,
+			"cleanup":[
+				"/app/bin/*"
+			],
+			"make-args":[
+				"PREFIX=/app"
+			],
+			"make-install-args":[
+				"PREFIX=/app"
+			],
+			"build-commands":[
+				"mkdir -p /app/share/applications",
+				"cp nvidia-settings.desktop /app/share/applications/nvidia-settings.desktop",
+				"mkdir -p /app/share/icons/hicolor/128x128/apps",
+				"cp nvidia-settings.png /app/share/icons/hicolor/128x128/apps/nvidia-settings.png"
+			],
+			"sources":[
+				{
+					"type":"git",
+					"url":"https://github.com/NVIDIA/nvidia-settings.git",
+					"branch":"@@NVIDIA_VERSION@@"
+				},
+				{
+					"type":"file",
+					"path":"nvidia-settings.desktop"
+				},
+				{
+					"type":"file",
+					"path":"nvidia-settings.png"
+				}
+			],
+			"modules":[
+				{
+					"name":"gtk",
```

Comment:
I tried it didn't work, initially it warns latter it fails. I will try deleting everything from gtk2 build and see what happens